src/components/linkCard/index.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './linkCard.module.scss';
+import ImageWrapper from '../imageWrapper';
 
-const LinkCard = ({ product, onClickFunction, sticky }) => {
+const LinkCard = ({ product, onClickFunction, sticky, imageResolution }) => {
   const style = sticky ? styles.sticky : styles.notSticky;
 
   return (
     <button onClick={() => onClickFunction(product.title)} className={style}>
-      <img src={product.image} alt={product.title} />
-      <div>
+      <ImageWrapper
+        src={product.image}
+        outerWrapperClassName={styles.imageContainer}
+        style={{ height: '100%', width: '100%' }}
```

Comment:
Burde ikke dette heller ligge i en klasse? Jeg ser dette er gjentagende for mange bilder nedover også.

Reply:
Skjønner ikke helt hva jeg skal skille ut til egen klasse, vi får ta det i morgen 🙂

Reply:
For eksempel:
```
.maxSize {
  height: 100%;
  width: 100%;
}
```

Hvis vi kan unngå `style` så gjør vi det der det er mulig :)

Reply:
Ah 👍

Reply:
Hvis det er det at den får to klasser du er bekymret over så kan du bruke `classNames` til det:
```js
<FooComponent className={classNames(class1, class2, class3)} />
```